app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java

```diff
@@ -19,10 +19,9 @@
     private CameraController mCameraController;
     private GoogleMap mMap;
```

Comment:
If you want consistency, mMap should be renamed aswell to either googleMap or mGoogleMap if you want to keep it a member variable. However, I prefer the latter since it is used throughout the class.

Reply:
how about if I keep the member variable name `mMap` here in MapInitializer.java, and instead rename the local variable `googleMap` in mapsActivity.java to `map`?
More consistent and a smaller change

Reply:
I have applied the change i mentioned above, let me know if you would rather the name convention you mentioned above instead.